Automattic/wp-calypso
# Reader: don't show 403/404 errors to user in stream

If a post is unavailable, don't show an error message in the stream. Fixes https://github.com/Automattic/wp-calypso/issues/11008.

Note: the original issue suggests a helpful message be displayed to site owners (and super admins) to potentially help debug the error. After discussion within #reader Slack channel, it was agreed that all users should see the same output.

Todo:
- [ ] fix <kbd>j</kbd>/<kbd>k</kbd> keyboard shortcuts

### To test

Try http://wpcalypso.wordpress.com/read/feeds/52282073. There is a trashed post at the beginning of the feed that displays an error message:

<img width="887" alt="screen shot 2017-01-27 at 17 44 16" src="https://cloud.githubusercontent.com/assets/17325/22381121/44b8db38-e4b8-11e6-8b20-aece1723aaac.png">

Now try http://calypso.localhost:3000/read/feeds/52282073. The trashed post at the beginning should no longer be visible:

<img width="877" alt="screen shot 2017-01-27 at 17 44 22" src="https://cloud.githubusercontent.com/assets/17325/22381127/4eecd618-e4b8-11e6-88df-dacb123fcf4f.png">
